### PR TITLE
Resolved issue #138,#137,#136,#135,#134

### PR DESCRIPTION
**Pull request by Garima-149**
@4darsh-Dev Please merge it and assign the labels as required.I tried to make 2 pr's but it comitted in single one.

### CODE REVIEW

File: django-web-app/templates/base.html [hunk 1]
```diff
@@ -103,13 +118,23 @@ <h1><a href="/"><img class="logo-img" src="{% static 'img/cogniguard-logo.png' %
                 <div class="side-items">
                     <i class="fas fa-search side-icons"></i>
                     <!-- <i class="fas fa-globe side-icons"></i> -->
+
+                    <a style="color: black;" target="_blank" href="https://github.com/4darsh-Dev/CogniGuard"><i
```
Comment (4darsh-Dev) [inline django-web-app/templates/base.html:122]: Here are two icons for github, remove one.

File: django-web-app/static/css/style.css [hunk 2]
```diff
@@ -1844,3 +1844,143 @@ footer {
...
+  border-radius: 50%;
+  background-color: var(--color-primary);
+  box-shadow: 0 2px 10px rgba(0, 0, 0, 0.1);
+}
+
+.chatbot-toggle:hover{
+  background-color: var(--dark-primary);
+}
+.chatbot-toggle .material-icons {
+  font-size: 40px; /* Adjust this size as needed */
```
Comment (4darsh-Dev) [inline django-web-app/static/css/style.css:1900]: much bigger icon , change to 32px

File: django-web-app/static/css/style.css [hunk 3]
```diff
@@ -1844,3 +1844,143 @@ footer {
...
+  border: none;
+  cursor: pointer;
+  color: white; /* Blue color */
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 60px;
+  height: 60px;
+  position: relative;
+  left:-170px ;
```
Comment (4darsh-Dev) [inline django-web-app/static/css/style.css:1889]: not visible, 
left : 80px ;

File: django-web-app/static/css/style.css [hunk 4]
```diff
@@ -1844,3 +1844,143 @@ footer {
...
+.chatbot {
+  display: none;
+  width: 300px;
+  background-color: var(--color-primary);
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+  border-radius: 10px;
+  overflow: hidden;
+  flex-direction: column;
+  position: absolute;
+  left:-170px ;
```
Comment (4darsh-Dev) [inline django-web-app/static/css/style.css:1912]: not visible on screen : 
use left:5rem;

**Review by 4darsh-Dev — CHANGES_REQUESTED** [review]
good work @Garima-149 ,
### Some changes are required. 
i've mentioned them in conversation

After fixing position issue, the preview I'm getting for chatbot. 
## SS
Screenshot 2024-07-19 192244

1. .message-bot background color is too light, make it little dark for proper visibility of text
2. It would be better if pressing enter will also send the message.

**Comment by Garima-149** [comment]
@4darsh-Dev I did these css changes.please check

**Review by 4darsh-Dev — APPROVED** [review]
okay there are still some code scanning warnings
i'll fix them later.

**Comment by 4darsh-Dev** [comment]
@Garima-149 , mark the issues as completed, 
PR is merged.

**Comment by Garima-149** [comment]
@4darsh-Dev I marked them already. 
Can I ask what levels you have assigned to issues I raised? Since, I made a single pr for all 5 issues.

**Comment by 4darsh-Dev** [comment]
there was only single pr for all issues so , I gave max level3 
only points are counted from PRs labels, i thnk.

**Comment by Garima-149** [comment]
@4darsh-Dev okay I didn't know it before, otherwise I would have made multiple ones.

**Comment by 4darsh-Dev** [comment]
not possible @Garima-149  now, 
it will look spammy, I have told you multiple times to not make single PR already, 
Will take care for the next times.

**Comment by 4darsh-Dev** [comment]
GSSoC 24 has been completed Finally'
 It was a great experience working with you all 
 Thanks💗 for your valuable contributions! 
 PA nomination has been started, Do fill out the forms soon. Share your experiences and let's connect on socials

**Pull request merged**